mac

```diff
@@ -125,7 +125,7 @@ brew link --force heroku
 
 fancy_echo "Configuring asdf version manager..."
 if [ ! -d "$HOME/.asdf" ]; then
-  git clone https://github.com/asdf-vm/asdf.git ~/.asdf --branch v0.3.0
```

Comment:
I feel like we should just be installing asdf via homebrew.

@tadp Is there a reason we did it this way?

Reply:
This was originally based off of the thoughtbot laptop script  which is platform agnostic.

 However, since we are all on Macs  I agree that using Homebrew  would make sense here